redis/src/main/scala/akka/streams/alpakka/redis/scaladsl/RedisSink.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.streams.alpakka.redis.scaladsl
+
+import akka.Done
+import akka.stream.scaladsl.{Keep, Sink}
+import akka.streams.alpakka.redis.{RedisHSet, RedisKeyValue, RedisKeyValues}
+import io.lettuce.core.api.StatefulRedisConnection
+import scala.collection.immutable.Seq
+import scala.concurrent.{ExecutionContext, Future}
+
+class RedisSink {
+
+  def set[K, V](parallelism: Int, connection: StatefulRedisConnection[K, V])(
+      implicit executionContext: ExecutionContext
+  ): Sink[RedisKeyValue[K, V], Future[Done]] =
+    RedisFlow.set(parallelism, connection).toMat(Sink.ignore)(Keep.right)
+
+  def apppend[K, V](parallelism: Int, connection: StatefulRedisConnection[K, V])(
+      implicit executionContext: ExecutionContext
+  ): Sink[RedisKeyValue[K, V], Future[Done]] =
+    RedisFlow.append(parallelism, connection).toMat(Sink.ignore)(Keep.right)
+
+  def mset[K, V](parallelism: Int, connection: StatefulRedisConnection[K, V])(
+      implicit executionContext: ExecutionContext
+  ): Sink[Seq[RedisKeyValue[K, V]], Future[Done]] =
+    RedisFlow.mset(parallelism, connection).toMat(Sink.ignore)(Keep.right)
+
+  def lpush[K, V](parallelism: Int, connection: StatefulRedisConnection[K, V])(
+      implicit executionContext: ExecutionContext
+  ): Sink[RedisKeyValues[K, V], Future[Done]] =
+    RedisFlow.lpush(parallelism, connection).toMat(Sink.ignore)(Keep.right)
+
+  def hset[K, V](parallelism: Int, connection: StatefulRedisConnection[K, V])(
+      implicit executionContext: ExecutionContext
+  ): Sink[RedisHSet[K, V], Future[Done]] =
+    RedisFlow.hset(parallelism, connection).toMat(Sink.ignore)(Keep.right)
+
+  def hmset[K, V](parallelism: Int, connection: StatefulRedisConnection[K, V])(
+      implicit executionContext: ExecutionContext
+  ): Sink[RedisHSet[K, V], Future[Done]] =
+    RedisFlow.hset(parallelism, connection).toMat(Sink.ignore)(Keep.right)
+
+}
```

Comment:
Do we actually need this file, or all the Sink signatures? Since all operations create a `RedisOperationResult` which contains the error indicators, using or providing this sink feels like adopting a norm of ignoring errors. No matter what happens, we all arrive at `Sink.ignore`.

Reply:
This is really good question and your comment makes sense but I prefer to give all possibles API for customer. So for example If user don't care about Redis operation result or don't mind  to lose some data in case master node is not available in the cluster, so he could use RedisSink instead of Flow. I did the same for Couchbase integration and mentioned this point in documentation.